sc2reader/resources.py

```diff
@@ -775,6 +775,11 @@ def register_default_readers(self):
             readers.GameEventsReader_65895(),
             lambda r: 65895 <= r.base_build,
```

Comment:
Can you update this line as well to have a max build to be in line with the others above?